lib/pavilion/output.py

```diff
@@ -70,29 +70,39 @@
     'UNDERLINE': UNDERLINE,
 }
 
+def format_duration(time_delta: float) -> str:
+    return str(datetime.timedelta(seconds=round(time_delta)))
 
-def get_relative_timestamp(base_time: float) -> str:
+def get_relative_timestamp(base_time: float,
+                           fullstamp: bool = False) -> str:
     """Print formatted time string based on the delta of time objects.
     :param base_time: The unix timestamp
+    :param bool fullstamp: Whether to return the full format timestamp.
     :returns: A formatted time string.
     """
 
     if not isinstance(base_time, float):
         return ''
 
     time_stamp_format = '%H:%M:%S'
-
-    now = datetime.datetime.now()
+    full_date_string = '%Y/%m/%d '
+    dayofweek = '%a '
+    monthday = '%b %d '
     base_time = datetime.datetime.fromtimestamp(base_time)
-    time_diff = now - base_time
-    if time_diff.days < 1 and now.date() == base_time.date():
-        format_ = time_stamp_format
-    elif time_diff.days < 7:
-        format_ = '%a ' + time_stamp_format
-    elif now.year == base_time.year:
-        format_ = '%b %d ' + time_stamp_format
+
+    if fullstamp:
+        format_ = full_date_string + time_stamp_format
```

Comment:
There's a more standard way to do this, with an actual standard - `.isoformat()` or `.isoformat(' ')`.
That's only an 'ok' format though - to do it properly you need a lot of libraries to handle time zones. Pavilion used to use those, but switched to straight unix time to be simpler. 

You don't have to change to this - just letting you know it exists.

Reply:
The best way I've found looks like this:
```
import pytz
import datetime

my_time = datetime.datetime.now()
# Usually this would be a per-system setting.
time_zone = pytz.timezone('US/Mountain')
# Localize the time based on the time zone
localized_ts = time_zone.localize(my_time)
# Now if you do .iso_format(), it will include timezone information. The resulting timezone should be (mostly) non-ambiguous.
print(localized_ts.isoformat())

Reply:
Oh yeah, I've seen that library.  For something so simple, it sure does cause a lot of dependency issues as far as I've seen.  

And as far as I can tell, that takes care of these issues here.